File: Modul 4/SOAL 4/PRAK304-2210817210002-GustiMuhammadNaufalRazin.py
for i in range(5):
    a = int(input())
    if a>=20 and a<=99 or a==10:
        print("Puluhan")
    elif a>99 or a<0:
        print("Anda Menginput Melebihi Limit Bilangan")
    elif a>=11 and a<=19:
        print("Belasan")
    elif a>=1 and a<=9:
        print("Satuan")
    else:
        print("Nol")

File: Modul 4/SOAL 4/PRAK304-2210817210002-GustiMuhammadNaufalRazin.c
#include <stdio.h>
int main()
{
    for(int i=0; i<=4; i++){
    int a;
    scanf("%d",&a);

    if(a>=1 && a<=9){
       printf("Satuan");}
    else if(a==10 || a>=20 && a<=99){
        printf("Puluhan");}
    else if(a>=11 && a<20){
        printf("Belasan");}
    else if(a>=100){
        printf("Anda Menginput Melebihi Limit Bilangan");}
    else if(a==0){
        printf("Nol");}

}
}
